src/compose-create-pull-request.ts

```diff
@@ -143,7 +143,7 @@ export async function composeCreatePullRequest(
     };
   } = await octokit.graphql(
     `
-    query ($owner: String!, $repo: String!, $head: String!) {
+    query PullRequestsForBranch($owner: String!, $repo: String!, $head: String!) {
```

Comment:
> I would call it `getPullRequestsForBranch`

I went with a noun-based name instead of verb-based and started with a capital letter since that seems to be common for naming queries. I'm not super well versed with this, so very open to feedback if you have other thoughts

Reply:
I'm not familiar with any such convention, do you have any resources that suggest it? I'm always happy to follow ecosystem conventions. For my projects so far I always named GraphQL queries like I would name JS functions

Reply:
I fully admit that I'm not strong in this area, so my observation of the pattern is weak at best, but here are a couple of examples i've come across in the last few days:
* https://graphql.org/learn/queries/#operation-type-and-name
* https://mswjs.io/docs/basics/intercepting-requests#graphql-request-matching
* https://www.apollographql.com/docs/react/data/operation-best-practices#name-all-operations

even in those few examples, the noun vs verb idea is in conflict, weakening my suggestion even further :) 

i dont have any reason to push back if you prefer a different name than my first suggestion

Reply:
I suppose I left this in an ambiguous state. Would you prefer I switch this to your original suggestion? Happy to go whichever direction you prefer

Reply:
These are great references though. If I started today, I'd probably prefer `PullRequestsForBranch`, but I think I'd prefer to stick with `getPullRequestsForBranch` now, at least for octokit(-adjacent) projects